.github/workflows/ci.yaml

```diff
@@ -101,10 +105,17 @@ jobs:
             cargo-cache-
 
       - name: Run command
+        if: ${{ github.ref != 'refs/heads/main' }}
         run: |
           ./scripts/docker_run ./scripts/runner --commands ${{ matrix.cmd }}
```

Comment:
I would add an explicit scope here, otherwise this will change meaning if / when the default is changed for that flag, right?

Reply:
Good point. Done.